watcher.go

```diff
@@ -175,3 +184,34 @@ func updatePromSDConfigMap(client *kubernetes.Clientset, fileSDConfig *v1.Config
 	}
 	return nil
 }
+
+func checkDuplicates(values []map[string][]string, value string) bool {
```

Comment:
nit: rename this `isDuplicate` and update the params to be clearer:

func isDuplicate(existing []map[string][]string, newTarget string) bool